doc/en/components/layouts/tile-manager.md

```diff
@@ -0,0 +1,316 @@
+| `bottom-adorner` | Overwrite the default vertical resize adorner. |
+
+
+### Header section actions
+
+By default, the header section includes two action buttons:
+
+- The `maximize` button enlarges the tile's content to fill the entire width of the Tile Manager, offering a wider view of the content.
+- The `fullscreen` button enables the tile to open in fullscreen mode in the user's browser.
+
+<img src="../../images/tile-manager-actions.png" />
+
+If you want to display just one of the two buttons, you can set it as a slot attribute within the Tile component. Use the `maximize-action` value to show only the maximize button, or the `fullscreen-action` value to show only the fullscreen button.
+
+```html
+<igc-tile-manager>
+  <igc-tile>
+    <div slot="maximize-action"></div>
+    <p>Content for Tile 1</p>
+  </igc-tile>
+</igc-tile-manager>
+```
+
+You also have the option to disable both action buttons and create custom ones according to your preferences.
+
+`sample="/layouts/tile-manager/actions", height="700", alt="{Platform} Tile Manager Actions Example"`
+
+In this example, we created custom action buttons using the Ignite UI Icon Button component.
+
+## Resizing
+
+Resizing in the tile manager is a functionality that allows tiles to be resized using three different resize adorners.
+
+- **Side Adorner**: Adjusts the width by modifying the column span.
+- **Bottom Adorner**: Adjusts the height by modifying the row span.
+- **Corner Adorner**: Adjusts both width and height simultaneously.
+
+To ensure smooth resizing, a ghost element is used instead of directly modifying the tile’s dimensions. This element appears on top of the original tile, displaying its current dimensions when resizing begins, and it updates in real time as the user drags any of the resize handles.
+
+> [!Note] If the ghost element exceeds the available grid space, it will automatically adjust to the largest possible span within the grid's limits.
+
+We can use the `resize-mode` property to control how resizing is applied in the Tile Manager. It can be set to either `hover` or `always`, which determines when the resize adorners are visible.
+
+```html
+<igc-tile-manager resize-mode='hover'>
+  <igc-tile>
+    <p>Tile 1</p>
+  </igc-tile>
+  <igc-tile>
+    <p>Tile 2</p>
+  </igc-tile>
+</igc-tile-manager>
+```
+
+You can see the difference between the two states in the example below:
+
+`sample="/layouts/tile-manager/resize", height="522", alt="{Platform} Tile Manager Resize Example"`
+
+### Limitations
+
+There are several constraints and limitations in the resizing process:
+
+- A tile cannot be resized smaller than its defined minimum width or height (minColWidth, minRowHeight).
+- A tile cannot exceed the maximum available space in the grid. If a tile starts at a specific column or row, and the user tries to resize it beyond the visible grid area, it will only expand up to the maximum available columns or rows from its starting position.
```

Comment:
_"A tile cannot exceed the maximum available space in the grid."_ - This restriction is valid for horizontal space only. 
_"If a tile starts at a specific column or row, and the user tries to resize it beyond the visible grid area, it will only expand up to the maximum available columns or rows from its starting position."_ - Since yesterday, we removed this restriction so now even tiles in the last column could be resized to the maximum column-span as long as there's room to resize to the right.